elasticsearch/Dockerfile

```diff
@@ -31,7 +31,8 @@ RUN rpm --import https://packages.elastic.co/GPG-KEY-elasticsearch && \
     yum clean all
 # initial boring elasticsearch.yml is just to enable plugin installation.
 # put the real config in place in a deployment with a ConfigMap.
-ADD elasticsearch.yml logging.yml /usr/share/elasticsearch/config/
+ADD elasticsearch.yml /usr/share/elasticsearch/config/
+ADD index_templates /usr/share/elasticsearch/config/index_templates
```

Comment:
Good to know. Interesting that this works fine for [ViaQ](https://github.com/ViaQ/docker-elasticsearch/pull/12) (/me confused 😖 ). I will fix it.